package/yast2-rdp.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Fri Aug 17 15:04:55 CEST 2018 - schubi@suse.de
+
+- Changed licence to GPL-2.0-only.
```

Comment:
hmm, this is confusing as you do not change license. Only its format in spec. So maybe something like "Switch license in spec from SPDX2 to SPDX3 format."